travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -63,5 +67,4 @@ ignore-docstrings=yes
 notes=
 
 [IMPORTS]
-deprecated-modules=pdb,pudb,ipdb
-
+deprecated-modules=openerp.osv
```

Comment:
@pedrobaeza 
This is not necesary, because is duplicate the same error in pr.cfg and stable.cfg
But I don't have problem in duplicate the same error then I fixed!
Thanks for you feedback!